web/admin/src/pages/categories/index.vue

```diff
@@ -272,14 +306,34 @@ export default defineComponent({
       }
     }
 
+    const handleClick = () => {
+      if (inputRef.value !== null) {
+        inputRef.value.click()
+      }
+    }
+
+    const handleInputFileChange = () => {
+      const files = inputRef.value?.files
+      if (inputRef.value && inputRef.value.files) {
```

Comment:
好みの問題だけど、アーリーリターンしてあげるとif分のネストが減らせていいかも

```suggestion
      unless (inputRef.value && inputRef.value.files) {
        return
      }
```

Reply:
どういうことだ、

Reply:
明日考える